src/utils/index.ts

```diff
@@ -0,0 +1,10 @@
+import { IVSCode } from '../models/IVSCode';
+
+export const vscode: IVSCode = {
+  env: { appName: 'Code' },
+  version: '1000.0.0',
```

Comment:
This is a mock for `vscode` module. Unfortunately, `vscode` methods can only be accessed while using Visual Studio Code (i.e. running tests through vscode itself). 

When we build the shipped `icons.json` by using `npm run build` we're not running into the Visual Studio Code process but node instead and thus, the process will fail to find `vscode` module.

The same happens when we run `mocha` directly in our CI environment. 

That's why I introduced the `IVSCode` interface to inject `vscode` module as a dependency in modules that do not need Visual Studio Code context.

Ideally, all the tests and processes should be executed in a Visual Studio Code context by running a VSCode process and run all through it but when dealing with node process you have to abstract `vscode` module away and inject it properly. This is also nice for testing purposes.

Note that there are some places that the injection of the `vscode` module is useless. That is what I commented you in Slack about `css` not supporting absolute paths in VSCode. I hope to have the time of writing a PR for this and send it to their Github repo. If it's not accepted then we'll remove this as it's really not needed at the moment but I'd rather to leave the code as the current solution is marked as a `temporary hack`.